.gitignore

```diff
@@ -1,4 +1,43 @@
-/vendor/*
+# CakePHP specific files #
+##########################
 /config/app.php
-/tmp/*
+/config/.env # josegonzalez/php-dotenv
 /logs/*
+/tmp/*
+/tmp/bake/*
```

Comment:
The deploy scripts live take care if it (or should!), and locally they are auto.-created.
No need IMO to do anything else here.